src/Sulu/Bundle/CustomUrlBundle/Controller/CustomUrlController.php

```diff
@@ -117,8 +117,7 @@ public function postAction($webspace, Request $request)
     {
         $document = $this->customUrlManager->create(
             $webspace,
-            $request->request->all(),
-            $this->getRequestParameter($request, 'targetLocale', true)
+            $request->request->all()
         );
```

Comment:
I think this should for bc reasons something like:

```suggestion
        $this->getRequestParameter($request, 'targetLocale', true);

        $document = $this->customUrlManager->create(
            $webspace,
            $request->request->all()
       );
```

to validate and throw a 400 error if targetLocale is not sent to the api. If we remove the getRequestParameter we will not get any validation about the required parameter.